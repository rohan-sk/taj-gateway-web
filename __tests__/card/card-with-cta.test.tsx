import { render } from "@testing-library/react";
import CardWithCta from "../../components/card/card-with-cta.component";

it("shows card with cta component", () => {
  const { asFragment } = render(<CardWithCta />);
  expect(asFragment()).toMatchSnapshot();
});
