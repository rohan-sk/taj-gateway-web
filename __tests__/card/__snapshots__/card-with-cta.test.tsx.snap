// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows card with cta component 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-1ogyrjv"
  >
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-cospxd-MuiPaper-root-MuiCard-root"
    />
    <div
      class="MuiBox-root css-h0xyzq"
    >
      <p
        class="MuiTypography-root MuiTypography-body1 css-1xoedcw-MuiTypography-root"
      />
      <p
        class="MuiTypography-root MuiTypography-body1 css-11eraoz-MuiTypography-root"
      />
    </div>
  </div>
</DocumentFragment>
`;
